pom.xml

```diff
@@ -53,7 +53,7 @@
    <properties>
        <!-- base url for site deployment.  See distribution management for full url.  Override this in settings.xml for staging -->
       <staging.siteURL>scp://people.apache.org/x1/www/activemq.apache.org</staging.siteURL>
-      <netty.version>4.0.28.Final</netty.version>
+      <netty.version>4.0.20.Final</netty.version>
```

Comment:
the conversation I was referring was the --allow-anonymous Y, I'm adding an option for N now


The downgrade.. I'm not sure I will merge it yet.. I'm trying to figure out why netty update broke the server.